apps/front-end/front-office/src/app/components/quiz-view/help-page/help-page.component.ts

```diff
@@ -1,8 +1,49 @@
-import { Component } from '@angular/core';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { Subject, takeUntil } from 'rxjs';
+import { Quiz } from '@webonjour/util-interface';
+import { selectGame } from '../../../reducers/game/game.selectors';
 
 @Component({
   selector: 'webonjour-help-page',
   templateUrl: './help-page.component.html',
   styleUrls: ['./help-page.component.scss'],
 })
-export class HelpPageComponent {}
+export class HelpPageComponent implements OnInit, OnDestroy {
+  randomClue!: Quiz.Clue;
+  quiz!: Quiz.Quiz;
+  currentQuizQuestion!: number;
+
+  public ngDestroyed$ = new Subject();
+
+  public ngOnDestroy() {
+    this.ngDestroyed$.next(0);
+  }
+
+  constructor(private store: Store) {}
+
+  ngOnInit() {
+    this.store
+      .select(selectGame)
+      .pipe(takeUntil(this.ngDestroyed$))
+      .subscribe((gameState) => {
+        if (gameState.quiz) {
+          this.quiz = gameState.quiz;
+          this.currentQuizQuestion = gameState.history.length;
+
+          const textClues = gameState.currentQuestion?.clues.filter(
+            (clue) => clue.text
+          );
+
+          if (textClues && textClues.length > 0) {
+            this.randomClue =
+              textClues[Math.floor(Math.random() * textClues.length)];
```

Comment:
Might want to put a comment to remind Math.random() < 1 so it cannot out of bounds

Reply:
https://github.com/2019-2020-ps6/2022-2023-ps6-webonjour/pull/211/commits/84ebc10ec02d0d1be9c6f63f75999fcbf9315d26